rector.php

```diff
@@ -16,7 +17,19 @@
         '*/Tests/*',
         '*.html.php',
         ContainerGetToConstructorInjectionRector::class => [
-            __DIR__.'/app/bundles/CoreBundle/Factory/MauticFactory.php', // Requires quite a refactoring.
+            __DIR__ .'/app/bundles/CoreBundle/Factory/MauticFactory.php', // Requires quite a refactoring.
+        ],
+
+        ReturnTypeFromReturnDirectArrayRector::class => [
+            // require bit test update
+            __DIR__.'/app/bundles/LeadBundle/Model/LeadModel.php',
+            // skip as a placeholder, not bool really
+            __DIR__.'/plugins/MauticCrmBundle/Integration/ZohoIntegration.php',
+        ],
+
+        \Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictBoolReturnExprRector::class => [
+            // skip as a placeholder, not bool really
+            __DIR__.'/app/bundles/LeadBundle/Segment/Decorator/BaseDecorator.php',
```

Comment:
Are these comments up to date?

Reply:
This PR is still WIP. Gotta run now.

I'll ping you once it's ready for review :+1: